package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hyperswitch",
-  "version": "0.8.8",
+  "version": "0.9.0",
```

Comment:
Do you know of any services? I don't. The only one that pops to mind is change prop, but it doesn't have incoming requests, so it isn't affected by it.

Yes, we definitely should start properly maintaining our packages :+1: